src/data/polynomial/lifts.lean

```diff
@@ -192,16 +192,18 @@ begin
   by_cases Rtrivial : nontrivial R,
```

Comment:
Hmm, maybe this also works:
```suggestion
  casesI subsingleton_or_nontrivial R with w w,
```

Reply:
Yeah it probably would, feel free to push the full change set if you want to do this. I was just trying to make the proof a bit more robust by not using the @ operator to supply TCs manually, not fully golf it

Reply:
Ok... I couldn't resist shaving off 10 lines...